ios/TravelSample/install_13.sh

```diff
@@ -5,7 +5,7 @@ cd Frameworks
 rm -rf iOS
 rm -rf macOS
 rm -rf tvOS
-curl https://packages.couchbase.com/releases/couchbase-lite-ios/3.0.0/couchbase-lite-swift_xc_enterprise_3.0.0.zip > cbl.zip
+curl https://packages.couchbase.com/releases/couchbase-lite-ios/3.0.2/couchbase-lite-swift_xc_enterprise_3.0.2co.zip > cbl.zip
```

Comment:
Are you sure the .zip file name is correct. There appears to be stray "co" at end

Reply:
...typo. will update